src/game/default/g_item.c

```diff
@@ -25,6 +25,10 @@
 const vec3_t ITEM_MINS = { -16.0, -16.0, -16.0 };
 const vec3_t ITEM_MAXS = { 16.0, 16.0, 16.0 };
 
+const g_armor_info_t bodyarmor_info = { 100, 200, 0.8, 0.6, ARMOR_BODY };
```

Comment:
I think I'd like to see these initialized into an array, rather than having 3 globals. Then G_GetArmorInfo could just loop over the array, looking for the tag.